### ISSUE TITLE
Support USFM3

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#835]
## Story

As a *user* I want to view the ULT and UST so that I can see what the say.

## Notes

We need to support USFM3 in our USFM to HTML convertor.

**Comment by RobH123** [comment]
It would seem that if you'd like a quick, temporary solution, extra \z fields and extra information in \w ...\w* could simply be dropped in the preprocessing stage of the Door43 Job Handler. I think this would instantly make both the existing tX Job Handler linters and converters work. Should be quick and easy in the short term -- although not of course a long-term solution. But your title here does say _basic_.

Alternatively if that's not a desired direction, perhaps the 2nd note above should say "support USFM3 in .. converter AND LINTER"???

**Comment by RobH123** [comment]
Seems that there are also other basic USFM errors, e.g., this faulty \q marker in Heb 1:5 line:

\w say|x-occurrence="2" x-occurrences="2"\w*,\q"\zaln-s | x-strong="G14730" x-lemma="ἐγώ" x-morph="Gr,RP,,,1N,S," x-occurrence="2" x-occurrences="2" x-content="ἐγὼ"

**Comment by RobH123** [comment]
Oh dear, there are also lots of whitespace problems in the ULT. How will the software know that the space should come BEFORE the " not after it at the EOL in Mat 17:22:

\w disciples|x-occurrence="1" x-occurrences="1"\w*,"\zaln-s | x-strong="G35880" x-lemma="ὁ" x-morph="Gr,EA,,,,NMS," x-occurrence="2" x-occurrences="2" x-content="ὁ"
\w The|x-occurrence="1" x-occurrences="1"\w*

**Comment by jag3773** [comment]
@RobH123 The whitespace issue is a known problem that is being fixed.  Internally, tC has the correct data but it is outputting the spaces incorrectly.  That and many other issues are expected to be fixed in https://github.com/unfoldingWord-dev/translationCore/issues/5051 .

Regarding `zaln`, it could be that we've misinterpreted the [milestone](https://ubsicap.github.io/usfm/milestones/index.html) pairs.  For example, from [Jude 1](https://git.door43.org/unfoldingWord/en_ult/src/branch/master/66-JUD.usfm#L12):

```
\zaln-s | x-strong="G24550" x-lemma="Ἰούδας" x-morph="Gr,N,,,,,NMS," x-occurrence="1" x-occurrences="1" x-content="Ἰούδας"
\w Jude|x-occurrence="1" x-occurrences="1"\w*
\zaln-e\*,
```

I think we've been assuming that the `zaln-e` is the closing for the tag (which it kind of is), without realizing that there should be a closing `\*` on the `zaln-s` too.  @RobH123 Could you make a comment on https://github.com/unfoldingWord-dev/translationCore/issues/5051 if this is a change that we need to make?

**Comment by RobH123** [comment]
Yes, according to http://ubsicap.github.io/usfm/milestones/index.html, tC is wrong. I've added a comment as you suggested.
